modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequestForm.testcase

```diff
@@ -21,33 +21,36 @@ definition {
 			EVPUtils.addEVPSite();
 		}
 
-		task ("Log as Employee user") {
+		task ("Login with the Employee and go to EVP Portal") {
```

Comment:
Please just update this task to: `"Login with the Employee user and go to EVP Portal"`